src/stars--0.css

```diff
@@ -0,0 +1,4 @@
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Redundant new file for each stars block, they have identical styles so move them to your style.css file and combine selectors using comma